### PR TITLE
[ADD] website_sale_b2x_alt_price: display B2{B,C} complementary price

### PR DESCRIPTION
**Pull request by yajo**
- If your website is B2B, display a smaller B2C price next to the main one. And viceversa.
- Toggle feature independently in list and product templates.
- Toggle feature per website.
- Supports live price recomputing.

@Tecnativa TT24410

### CODE REVIEW

**Comment by sergio-teruel** [comment]
@Yajo fix travis

**Comment by yajo** [comment]
I'd say it's some kind of conflict with other module. I isolated this one's test. Let's see.

**Comment by yajo** [comment]
There was a problem in tests: website discounts were disabled. It should go :heavy_check_mark: now.

**Review by HaraldPanten — APPROVED** [review]
IMO, It's functionally working as expected 👍

File: website_sale_b2x_alt_price/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "maintainers": ["Yajo"],
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "website_sale",
+    ],
+    "data": [
+        # "security/some_model_security.xml",
```
Comment (chienandalu) [inline website_sale_b2x_alt_price/__manifest__.py]: :fire:

File: website_sale_b2x_alt_price/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+#. module: website_sale_b2x_alt_price
+#: model:ir.model,name:website_sale_b2x_alt_price.model_product_template
+msgid "Product Template"
+msgstr "Plantilla de producto"
+
+#. module: website_sale_b2x_alt_price
+#: model_terms:ir.ui.view,arch_db:website_sale_b2x_alt_price.alt_price
+msgid "with taxes"
+msgstr "con impuestos"
```
Comment (chienandalu) [inline website_sale_b2x_alt_price/i18n/es.po:28]: In a spanish e-commerce I'd except "IVA inc." o "excl. IVA". What do you think @pedrobaeza ?

**Review by chienandalu — CHANGES_REQUESTED** [review]
Works ok. Just some tiny remarks:

File: website_sale_b2x_alt_price/i18n/es.po (hunk 2 above)
Comment (pedrobaeza) [reply website_sale_b2x_alt_price/i18n/es.po:28]: "IVA incluido" is not accurated enough, as there can be other taxes that are not IVA, so I think this one is ok.

Comment (chienandalu) [reply website_sale_b2x_alt_price/i18n/es.po:28]: Ok, I didn't think of b2b scenarios

File: website_sale_b2x_alt_price/__manifest__.py (hunk 1 above)
Comment (yajo) [reply website_sale_b2x_alt_price/__manifest__.py]: Thanks, fixed. Could you please update your review?

**Review by chienandalu — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by HaraldPanten** [comment]
Just to know...

Why did you include a translation .po to Spanish? shouldn't it be managed by using Weblate?

**Comment by pedrobaeza** [comment]
It's not needed to wait for the translation till this is merged when adding a new module, as there's no possibility of conflict.

It's not advised to include translations in a `[IMP]` or `[FIX]` PR, because if there's already a translation for the module, there are chances that a conflict arises, but anyway, it's the duty of the PR creator to rebase and fix the conflict if still wants to include the translation.

What is forbidden and PRs are closed is a PR with only translations.

**Comment by HaraldPanten** [comment]
> It's not needed to wait for the translation till this is merged when adding a new module, as there's no possibility of conflict.
> 
> It's not advised to include translations in an `[IMP]` or `[FIX]`, because if there's already a translation for the module, there are chances that a conflict arises, but anyway, it's the duty of the PR creator to rebase and fix the conflict if still wants to include the translation.
> 
> What is forbidden and PRs are closed is a PR with only translations.

Thanks Pedro,

Didn't know that guideline. Much more clear now. 👍

**Comment by pedrobaeza** [comment]
Not sure why it's failing. @Yajo can you rebase and check?

**Review by mamcode — APPROVED** [review]
LGTM, Thanks.

**Comment by yajo** [comment]
It seems some sort of problem with website_tax_toggle tour.

**Comment by yajo** [comment]
The fix is in https://github.com/OCA/e-commerce/pull/423.

**Pull request merged**